pkgs/tools/admin/pulumi/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "pulumi-${version}";
+  version = "0.16.2";
+
+  pulumiArchPackage = {
+    "x86_64-linux" = {
+      url = "https://get.pulumi.com/releases/sdk/pulumi-v${version}-linux-x64.tar.gz";
```

Comment:
Can you add a comment as to where you found these links, for easier upgrade when a new version comes out? I'm a bit surprised it doesn't support `aarch64-linux`, but why not, I guess.

Reply:
crazy spa.. you have to switch the dropdown to manual: https://pulumi.io/quickstart/install.html

Reply:
i initially tried to build form source.. but it got too painful after having to deal with `yarn` + `pip` in addition to `go`

Reply:
Can you add the `switch the dropdown to “manual” on https://pulumi.io/quickstart/install.html` comment to the source? Especially as it's unintuitive :)

As for the fact it's not built from source, then `pulumi` should be replaced with `pulumi-bin` in `all-packages.nix`, as the convention appears to be on nixpkgs for packages that are not built from source while they could.

Reply:
i see, ill soon get back to it in more detail when sanitizing our k8s config (hope this one will do it)... wanna add update script and also need providers, was thinking to do it similar to terraform once i got time